imu:
  ros__parameters:
    i2c_bus: 1
    i2c_address: 0x68
    publish_rate: 50
    frame_id: "imu_link"
    topic_name: "/imu/data"
    calibration_file: "imu_calibration.yaml"
    enable_magnetometer: true
    enable_temperature: true

sensor_fusion:
  ros__parameters:
    algorithm: "madgwick"
    sample_frequency: 50
    beta: 0.1
    zeta: 0.003
    gain: 0.1
